src/Apps/Collect/Components/Header/index.tsx

```diff
@@ -42,13 +65,18 @@ export class CollectionHeader extends Component {
   }
 }
 
-const Background = styled(Box)`
+const Background = styled(Box).attrs<{ headerImageUrl: string }>({})`
```

Comment:
Styled components recently updated to support  

```jsx
const Background = styled(Box)<{ headerImageUrl: string }>`
  ...
`
```

Reply:
Nice!